src/types/systemIntake.ts

```diff
@@ -63,3 +68,5 @@ export type StoreSystemIntakesAction = {
   type: string;
   systemIntakes: SystemIntakeForm[];
 };
+
+export type SystemIntakeID = string;
```

Comment:
This is unnecessary. It can be marked as a `string` rather than creating it's own type alias.